# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by scripts/Create-CopyPatch.
# 
# T2 SDE: package/*/bash/getrandom.patch
# Copyright (C) 2021 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- bash-5.1/lib/sh/random.c.vanilla	2021-02-09 18:29:41.857662161 +0100
+++ bash-5.1/lib/sh/random.c	2021-02-09 18:30:18.967664076 +0100
@@ -184,7 +184,7 @@
 #  define GRND_RANDOM 2
 #endif
 
-static ssize_t
+ssize_t
 getrandom (buf, len, flags)
      void *buf;
      size_t len;
